<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DefinedLaser</class>
 <widget class="QWidget" name="DefinedLaser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>214</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QRadioButton" name="freqDub">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Frequency doubling</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="num_W">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>60</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="numDigits">
    <number>8</number>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_W">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:12pt; font-style:italic;&quot;&gt;W &lt;/span&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="num_lambda">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>80</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="numDigits">
    <number>8</number>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_lambda">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:12pt; font-style:italic;&quot;&gt;λ &lt;/span&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="num_frep">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>140</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="numDigits">
    <number>8</number>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_frep">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:12pt; font-style:italic;&quot;&gt;f&lt;/span&gt;&lt;span style=&quot; font-size:12pt; font-style:italic; vertical-align:sub;&quot;&gt;rep&lt;/span&gt;&lt;span style=&quot; font-size:12pt; font-style:italic;&quot;/&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="num_Cb">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>120</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="numDigits">
    <number>8</number>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_Cb">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:12pt; font-style:italic;&quot;&gt;C&lt;/span&gt;&lt;span style=&quot; font-size:12pt; font-style:italic; vertical-align:sub;&quot;&gt;b&lt;/span&gt;&lt;span style=&quot; font-size:12pt; font-style:italic;&quot;/&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="num_M2">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>100</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="numDigits">
    <number>8</number>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_M2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:12pt; font-style:italic;&quot;&gt;M&lt;/span&gt;&lt;span style=&quot; font-size:12pt; font-style:italic; vertical-align:super;&quot;&gt;2&lt;/span&gt;&lt;span style=&quot; font-size:12pt; font-style:italic;&quot;/&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="num_tau">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>160</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="numDigits">
    <number>8</number>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_tau">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:12pt; font-style:italic;&quot;&gt;τ &lt;/span&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="num_P">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>40</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="numDigits">
    <number>8</number>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_P">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:12pt; font-style:italic;&quot;&gt;P &lt;/span&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QSlider" name="slide_frep">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>142</y>
     <width>160</width>
     <height>18</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="tracking">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="slide_tau">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>162</y>
     <width>160</width>
     <height>18</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="tracking">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="slide_T">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>182</y>
     <width>160</width>
     <height>18</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="tracking">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_T">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>180</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:12pt; font-style:italic;&quot;&gt;T &lt;/span&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="num_T">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>180</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="numDigits">
    <number>8</number>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
  </widget>
  <widget class="QSlider" name="slide_Cb">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>122</y>
     <width>160</width>
     <height>18</height>
    </rect>
   </property>
   <property name="tracking">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="slide_M2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>102</y>
     <width>160</width>
     <height>18</height>
    </rect>
   </property>
   <property name="tracking">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="slide_P">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>42</y>
     <width>160</width>
     <height>18</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="tracking">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="slide_W">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>62</y>
     <width>160</width>
     <height>18</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="tracking">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="slide_lambda">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>82</y>
     <width>160</width>
     <height>18</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="tracking">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="unit_P">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>39</y>
     <width>48</width>
     <height>22</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="unit_W">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>59</y>
     <width>48</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="unit_lambda">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>79</y>
     <width>48</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="unit_frep">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>139</y>
     <width>48</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="unit_tau">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>159</y>
     <width>48</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="unit_T">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>179</y>
     <width>48</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>freqDub</tabstop>
  <tabstop>unit_P</tabstop>
  <tabstop>slide_P</tabstop>
  <tabstop>unit_W</tabstop>
  <tabstop>slide_W</tabstop>
  <tabstop>unit_lambda</tabstop>
  <tabstop>slide_lambda</tabstop>
  <tabstop>slide_M2</tabstop>
  <tabstop>slide_Cb</tabstop>
  <tabstop>unit_frep</tabstop>
  <tabstop>slide_frep</tabstop>
  <tabstop>unit_tau</tabstop>
  <tabstop>slide_tau</tabstop>
  <tabstop>unit_T</tabstop>
  <tabstop>slide_T</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
